shared-ui/src/main/res/layout/vault_actions_sheet_layout.xml

```diff
@@ -5,107 +5,135 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:padding="24dp"
-    android:background="@drawable/drag_anchor_drawable"
-    >
+    android:background="@drawable/drag_anchor_drawable">
+
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
```

Comment:
We can standardize all the changes into a common style for all the `TextView` components, that extends from `Tella_Main_White_Text.Meduim`